eth/stagedsync/stage_hashstate.go

```diff
@@ -263,14 +238,9 @@ func readPlainStateOnce(
 			if additionalLogArguments != nil {
 				logArs = append(logArs, additionalLogArguments(k, v)...)
 			} else {
-				// makeCurrentKeyStr is declared in erigon-lib/etl.go as private. That's why I have to redeclare it.
 				makeCurrentKeyStr := func(k []byte) string {
```

Comment:
I guess `makeCurrentKeyStr` will have 2 cases: 

- first when key has 4 leading zeroes, we save whole byte slice `k`
- otherwise we save only first 4 bytes of byte slice `k`

Reply:
or just always print 6 bytes (or 8 or 10).